cdap-app-templates/cdap-etl/cdap-etl-lib/src/test/java/co/cask/cdap/templates/etl/transforms/StructuredRecordToCubeFactTransformTest.java

```diff
@@ -199,8 +199,8 @@ public void testTransform() throws Exception {
     transformed = emitter.getEmitted().get(0);
     // note: our date format doesn't include millis
```

Comment:
we can remove now. Thogh still relevant - the date format used in test doesn't have millis